client/python/armada_client/client.py

```diff
@@ -224,6 +224,26 @@ def update_queue(
         )
         self.submit_stub.UpdateQueue(request)
 
+    def send_create_queue_request(self, queue: submit_pb2.Queue) -> empty_pb2.Empty:
+        """
+        Uses the CreateQueue RPC to create a queue.
+
+        :param queues: A list of queues to create.
+        """
+
+        response = self.submit_stub.CreateQueue(queue)
+        return response
+
+    def send_update_queue_request(self, queue: submit_pb2.Queue) -> empty_pb2.Empty:
+        """
+        Uses the CreateQueue RPC to update a queue.
```

Comment:
Comment should be updated for Update Queue

Reply:
Same with this